src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -75,4 +77,7 @@ object Instance {
     def isDropped: Boolean = instance.state.status == InstanceStatus.Dropped
   }
 
+  case class LaunchedEphemeral(
+    instance: Instance,
+    hostPorts: Seq[Int])
```

Comment:
because Task.LaunchEphemeral has them too, and they're eventually passed to Task.launched(), and from there ... used throughout the system to determine which ports were allocated for the task